src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Tag as AntTag } from 'antd'
+import { ReactElement } from 'react'
+
+import { TagProps } from './Tag.props'
+
+export const defaults = {
+  isBordered: true,
+}
+
+export const Tag = (
+  {
+    children,
+    color,
+    closeIcon,
+    onClose,
+    isBordered = defaults.isBordered,
+  } : TagProps
+) : ReactElement => {
+  return (
+    <AntTag
+      bordered={isBordered}
+      closeIcon={closeIcon}
+      color={color}
+      onClose={onClose}
+    >
+      {children}
```

Comment:
I guess we are going to use it most of the time with our `FormattedMessage` component but a full ReactNode in the tag could be abused 🤔  would it make sense to enforce a Typography element or to enforce a prop definition compatbile with our typography?

Reply:
I think it is important to ask the designer to know if the tag component will have in the future a specific size. But for i18n I suggest using a children's node